packages/client/src/metricSources/Prometheus/PrometheusMetricModal.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { AbstractMetricModal } from '../../components/config/AbstractMetricModal';
+import { PROMETHEUS_SERVICE_TYPE } from './index';
+import { InlineTextGroup } from '../../layout/InlineTextGroup';
+import { validateCanaryMetricConfig } from '../../validation/configValidators';
+import { CanaryMetricConfig } from '../../domain/Kayenta';
+import { ValidationResultsWrapper } from '../../domain/Referee';
+import PrometheusCanaryMetricSetQueryConfig from './PrometheusCanaryMetricSetQueryConfig';
+
+export default class PrometheusMetricModal extends AbstractMetricModal<PrometheusCanaryMetricSetQueryConfig> {
+  validateCanaryMetricConfig(existingMetric: CanaryMetricConfig, type: string): ValidationResultsWrapper {
+    return validateCanaryMetricConfig(existingMetric, type, true);
+  }
+
+  getQueryInitialState(): PrometheusCanaryMetricSetQueryConfig {
+    return {
+      type: PROMETHEUS_SERVICE_TYPE,
+      customInlineTemplate: ''
+    };
+  }
+
+  getMetricSourceSpecificJsx(): JSX.Element {
+    return (
+      <div>
+        <InlineTextGroup
+          onBlur={() => {
+            this.touch('customInlineTemplate');
+          }}
+          touched={this.state.touched.customInlineTemplate}
+          error={this.state.errors['query.customInlineTemplate']}
+          id="customInlineTemplate"
+          label="Custom Inline Template"
+          value={this.state.metric.query.customInlineTemplate}
+          onChange={e => this.updateQueryObject('customInlineTemplate', e.target.value)}
+          placeHolderText="IDK WHAT THIS IS LMAO"
+          subText="boy i sure hope this works"
```

Comment:
This should be help text that shows up under the text box.

I see though that you are implementing this slightly different than we did with SignalFx so probably need to fire this up locally to sanity check.

https://github.com/Nike-Inc/referee/blob/master/packages/client/src/metricSources/SignalFx/SignalFxMetricModal.tsx#L173-L205